Model/MenuRepository.php

```diff
@@ -149,4 +150,16 @@ public function get($identifier, $storeId)
         $collection->addFilter('store_id', $storeId);
         return $collection->getFirstItem();
     }
+
+    public function setIsActiveByIds($ids, $isActive)
+    {
+        /** @var Collection $collection */
+        $collection = $this->collectionFactory->create();
+        $connection = $collection->getConnection();
+        $connection->update(
```

Comment:
this should be a part of menu resource model

Reply:
As discussed, we'll keep it like this as collection is still part of a resource model and here changes are made to a collection of menus.